src/main/java/com/backbase/ct/bbfuel/data/ContactsDataGenerator.java

```diff
@@ -51,32 +51,9 @@ private static ExternalContact generateContact(int numberOfAccounts) {
         List<ExternalAccountInformation> accounts = new ArrayList<>();
 
         for (int i = 0; i < numberOfAccounts; i++) {
-            ExternalAccountInformation externalAccountInformation = new ExternalAccountInformation()
-                .externalId(UUID.randomUUID().toString().substring(0, 32))
-                .name(faker.lorem().sentence(3, 0).replace(".", ""))
-                .alias(faker.lorem().characters(10))
-                .bic(VALID_BIC_LIST.get(faker.random().nextInt(VALID_BIC_LIST.size())))
-                .accountHolderAddress(new Address()
-                    .addressLine1(faker.address().streetAddress())
-                    .addressLine2(faker.address().secondaryAddress())
-                    .streetName(faker.address().streetAddress())
-                    .postCode(faker.address().zipCode())
-                    .town(faker.address().cityName())
-                    .country(faker.address().countryCode())
-                    .countrySubDivision(faker.address().state()))
-                .accountType(VALID_ACCOUNT_TYPE_LIST.get(faker.random().nextInt(VALID_ACCOUNT_TYPE_LIST.size())))
-                .bankCode(createRandomValidRtn())
-                .bankAddress(new Address()
-                    .addressLine1(faker.address().streetAddress())
-                    .addressLine2(faker.address().secondaryAddress())
-                    .streetName(faker.address().streetAddress())
-                    .postCode(faker.address().zipCode())
-                    .town(faker.address().city())
-                    .country(faker.address().countryCode())
-                    .countrySubDivision(faker.address().state()));
-
-            externalAccountInformation = determineTheUseOfIBANorBBAN(externalAccountInformation);
-
+            ExternalAccountInformation externalAccountInformation = faker.random().nextInt(3) % 2 == 0 ?
```

Comment:
I suggest to use `.random().nextInt(0, 1)` instead of `.random().nextInt(3) % 2 == 0`

Reply:
changed to  `faker.random().nextBoolean()`